Packages/pargap.mixer

```diff
@@ -3,7 +3,7 @@
 <mixertitle><mixer var="GAP"/> package ParGAP</mixertitle>
 
 <h2>Parallel GAP</h2>
-<p class="homelink">[<a href="http://www.gap-system.org/HostedGapPackages/pargap/">WWW homepage</a>]</p>
+<p class="homelink">[<a href="https://www.gap-system.org/HostedGapPackages/pargap/">WWW homepage</a>]</p>
```

Comment:
The majority of files called `Packages/package-name.mixer` are generated from the information in their `PackageInfo.g` (except for packages that are no longer redistributed with GAP but have a page for the reference purposes). So while we can merge this, it might be overwritten next time.